server/models/Request.js

```diff
@@ -0,0 +1,20 @@
+import mongoose from "mongoose";
+
+const { Schema, model } = mongoose;
+
+const requestSchema = new mongoose.Schema({
+  customer: { type: Schema.Types.ObjectId, ref: "User", required: true },
+  provider: { type: Schema.Types.ObjectId, ref: "User", required: true },
+  start: { type: Date, required: true },
+  end: { type: Date, required: true },
+  accepted: { type: Boolean, default: false },
+  declined: { type: Boolean, default: false },
+  paid: { type: Boolean, default: false },
+  serviceType: {
+    type: String,
+    enum: ["Boarding", "House Sitting", "Day Care", "Walking"],
+  },
+  rating: { type: Number },
+});
```

Comment:
You can add a `timestamps` option here to let Mongoose automatically manage `createdAt` and `updatedAt` properties on the documents. Check it out [here](https://mongoosejs.com/docs/guide.html#timestamps)